ocf_datapipes/training/common.py

```diff
@@ -37,6 +37,36 @@
 logger = logging.getLogger(__name__)
 
 
+def is_config_and_path_valid(
+    use_flag: bool,
+    config: Optional[InputData],
```

Comment:
@roshnaeem my bad didn't check this properly last night, the type I suggested is the wrong type, since it's the various attributes of the config input which is actually passed in (like PV, Satellite etc) not the InputData itself, the type hint would be quite long here so I think this type hint can just be removed for now actually, sorry about that